BTCPayServer/wwwroot/swagger/v1/swagger.template.stores-wallet.on-chain.json

```
@@ -382,7 +382,7 @@
                 "tags": [
                     "Store Wallet (On Chain)"
                 ],
-                "summary": "Get store on-chain wallet transactions",
+                "summary": "Get a store on-chain wallet transaction",
```

```suggestion
                "summary": "Get store on-chain wallet transaction",
```
just to stay consistent